scripts/install-cni.sh

```diff
@@ -95,7 +95,13 @@ else
 fi
 
 token=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
-node_url="https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}/api/v1/nodes/${HOSTNAME}"
+host=${KUBERNETES_SERVICE_HOST}
+# If host constains a colon (:), it is an IPv6 address, hence needs wrapping
+# with [..].
+if [ "${host#*:}" != "$host" ]; then
```

Comment:
can we say `[[ "${host}" =~ : ]]` instead? `[ "${host#*:}" != "$host" ]` is a bit obscured.

Reply:
Thanks for the suggestion - done.